docs/index.yml

```diff
@@ -38,6 +38,12 @@ landingContent:
             url: project/index.yml
           - text: Word add-ins documentation
             url: word/index.yml
+      - linkListType: learn
```

Comment:
Maybe we can move this learn section higher - place after concept?

Reply:
I like that idea! All of the documentation links under Overview are also available in the sidebar, so I don't think we lose much by moving them down.